<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hadesvm::gui::MainWindow</class>
 <widget class="QMainWindow" name="hadesvm::gui::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>266</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HADES VM</string>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <property name="spacing">
     <number>8</number>
    </property>
    <item row="7" column="1" colspan="2">
     <widget class="QPushButton" name="configureVmPushButton">
      <property name="text">
       <string>Configure</string>
      </property>
      <property name="icon">
       <iconset resource="Resources.qrc">
        <normaloff>:/hadesvm/gui/images/actions/ConfigureSmall.png</normaloff>:/hadesvm/gui/images/actions/ConfigureSmall.png</iconset>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QPushButton" name="suspendPushButton">
      <property name="text">
       <string>Suspend</string>
      </property>
      <property name="icon">
       <iconset resource="Resources.qrc">
        <normaloff>:/hadesvm/gui/images/actions/SuspendSmall.png</normaloff>:/hadesvm/gui/images/actions/SuspendSmall.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="9">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QPushButton" name="stopVmPushButton">
      <property name="text">
       <string>Stop</string>
      </property>
      <property name="icon">
       <iconset resource="Resources.qrc">
        <normaloff>:/hadesvm/gui/images/actions/StopSmall.png</normaloff>:/hadesvm/gui/images/actions/StopSmall.png</iconset>
      </property>
     </widget>
    </item>
    <item row="4" column="1" rowspan="2" colspan="2">
     <widget class="QPushButton" name="resumePushButton">
      <property name="text">
       <string>Resume</string>
      </property>
      <property name="icon">
       <iconset resource="Resources.qrc">
        <normaloff>:/hadesvm/gui/images/actions/ResumeSmall.png</normaloff>:/hadesvm/gui/images/actions/ResumeSmall.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QPushButton" name="startVmPushButton">
      <property name="text">
       <string>Start</string>
      </property>
      <property name="icon">
       <iconset resource="Resources.qrc">
        <normaloff>:/hadesvm/gui/images/actions/StartSmall.png</normaloff>:/hadesvm/gui/images/actions/StartSmall.png</iconset>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>118</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>8</width>
        <height>8</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="1">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>8</width>
        <height>8</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>408</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNewVm"/>
    <addaction name="actionOpenVm"/>
    <addaction name="separator"/>
    <addaction name="actionCloseVm"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuVm">
    <property name="title">
     <string>&amp;VM</string>
    </property>
    <addaction name="actionStartVm"/>
    <addaction name="actionStopVm"/>
    <addaction name="separator"/>
    <addaction name="actionSuspendVm"/>
    <addaction name="actionResumeVm"/>
    <addaction name="separator"/>
    <addaction name="actionResetVm"/>
    <addaction name="separator"/>
    <addaction name="actionConfigureVm"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuVm"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/hadesvm/gui/images/actions/ExitSmall.png</normaloff>:/hadesvm/gui/images/actions/ExitSmall.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionStartVm">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/hadesvm/gui/images/actions/StartSmall.png</normaloff>:/hadesvm/gui/images/actions/StartSmall.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Start</string>
   </property>
  </action>
  <action name="actionStopVm">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/hadesvm/gui/images/actions/StopSmall.png</normaloff>:/hadesvm/gui/images/actions/StopSmall.png</iconset>
   </property>
   <property name="text">
    <string>S&amp;top</string>
   </property>
  </action>
  <action name="actionConfigureVm">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/hadesvm/gui/images/actions/ConfigureSmall.png</normaloff>:/hadesvm/gui/images/actions/ConfigureSmall.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/hadesvm/gui/images/actions/AboutSmall.png</normaloff>:/hadesvm/gui/images/actions/AboutSmall.png</iconset>
   </property>
   <property name="text">
    <string>A&amp;bout...</string>
   </property>
  </action>
  <action name="actionNewVm">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/hadesvm/gui/images/actions/NewSmall.png</normaloff>:/hadesvm/gui/images/actions/NewSmall.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New VM</string>
   </property>
  </action>
  <action name="actionOpenVm">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/hadesvm/gui/images/actions/OpenSmall.png</normaloff>:/hadesvm/gui/images/actions/OpenSmall.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open VM</string>
   </property>
  </action>
  <action name="actionCloseVm">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/hadesvm/gui/images/actions/CloseSmall.png</normaloff>:/hadesvm/gui/images/actions/CloseSmall.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;lose VM</string>
   </property>
  </action>
  <action name="actionSuspendVm">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/hadesvm/gui/images/actions/SuspendSmall.png</normaloff>:/hadesvm/gui/images/actions/SuspendSmall.png</iconset>
   </property>
   <property name="text">
    <string>S&amp;uspend</string>
   </property>
  </action>
  <action name="actionResumeVm">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/hadesvm/gui/images/actions/ResumeSmall.png</normaloff>:/hadesvm/gui/images/actions/ResumeSmall.png</iconset>
   </property>
   <property name="text">
    <string>R&amp;esume</string>
   </property>
  </action>
  <action name="actionResetVm">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/hadesvm/gui/images/actions/ResetSmall.png</normaloff>:/hadesvm/gui/images/actions/ResetSmall.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Reset</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/hadesvm/gui/images/actions/PreferencesSmall.png</normaloff>:/hadesvm/gui/images/actions/PreferencesSmall.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
   <property name="iconText">
    <string>Preferences</string>
   </property>
   <property name="toolTip">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionNewVm</sender>
   <signal>triggered()</signal>
   <receiver>hadesvm::gui::MainWindow</receiver>
   <slot>_onNewVm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpenVm</sender>
   <signal>triggered()</signal>
   <receiver>hadesvm::gui::MainWindow</receiver>
   <slot>_onOpenVm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCloseVm</sender>
   <signal>triggered()</signal>
   <receiver>hadesvm::gui::MainWindow</receiver>
   <slot>_onCloseVm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>hadesvm::gui::MainWindow</receiver>
   <slot>_onExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStartVm</sender>
   <signal>triggered()</signal>
   <receiver>hadesvm::gui::MainWindow</receiver>
   <slot>_onStartVm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStopVm</sender>
   <signal>triggered()</signal>
   <receiver>hadesvm::gui::MainWindow</receiver>
   <slot>_onStopVm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionConfigureVm</sender>
   <signal>triggered()</signal>
   <receiver>hadesvm::gui::MainWindow</receiver>
   <slot>_onConfigureVm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>hadesvm::gui::MainWindow</receiver>
   <slot>_onHelpAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startVmPushButton</sender>
   <signal>clicked()</signal>
   <receiver>hadesvm::gui::MainWindow</receiver>
   <slot>_onStartVm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopVmPushButton</sender>
   <signal>clicked()</signal>
   <receiver>hadesvm::gui::MainWindow</receiver>
   <slot>_onStopVm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>configureVmPushButton</sender>
   <signal>clicked()</signal>
   <receiver>hadesvm::gui::MainWindow</receiver>
   <slot>_onConfigureVm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSuspendVm</sender>
   <signal>triggered()</signal>
   <receiver>hadesvm::gui::MainWindow</receiver>
   <slot>_onSuspendVm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionResumeVm</sender>
   <signal>triggered()</signal>
   <receiver>hadesvm::gui::MainWindow</receiver>
   <slot>_onResumeVm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>suspendPushButton</sender>
   <signal>clicked()</signal>
   <receiver>hadesvm::gui::MainWindow</receiver>
   <slot>_onSuspendVm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resumePushButton</sender>
   <signal>clicked()</signal>
   <receiver>hadesvm::gui::MainWindow</receiver>
   <slot>_onResumeVm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>hadesvm::gui::MainWindow</receiver>
   <slot>_onCurrentVmChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionResetVm</sender>
   <signal>triggered()</signal>
   <receiver>hadesvm::gui::MainWindow</receiver>
   <slot>_onResetVm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPreferences</sender>
   <signal>triggered()</signal>
   <receiver>hadesvm::gui::MainWindow</receiver>
   <slot>_onPreferences()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_onNewVm()</slot>
  <slot>_onOpenVm()</slot>
  <slot>_onCloseVm()</slot>
  <slot>_onExit()</slot>
  <slot>_onStartVm()</slot>
  <slot>_onStopVm()</slot>
  <slot>_onConfigureVm()</slot>
  <slot>_onHelpAbout()</slot>
  <slot>_onSuspendVm()</slot>
  <slot>_onResumeVm()</slot>
  <slot>_onCurrentVmChanged(int)</slot>
  <slot>_onResetVm()</slot>
  <slot>_onPreferences()</slot>
 </slots>
</ui>
